test/runtests.jl

```diff
@@ -37,5 +37,7 @@ end
         @time @safetestset "Symbol and integer based indexing of integrators" begin include("downstream/integrator_indexing.jl") end
         @time @safetestset "Unitful interoplations" begin include("downstream/unitful_interpolations.jl") end
         @time @safetestset "Integer idxs" begin include("downstream/integer_idxs.jl") end
+        # @time @safetestset "Autodiff Remake" begin include("downstream/remake_autodiff.jl") end
```

Comment:
what happened to this one?

Reply:
No idea, wasn't in the tests. I added it but commented out

Reply:
add it?